molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java

```diff
@@ -139,7 +142,7 @@ protected void configure(HttpSecurity http) throws Exception
 
 		.formLogin().loginPage("/login").failureUrl("/login?error").and()
 
-		.logout().addLogoutHandler((req, res, auth) -> {
+		.logout().deleteCookies("JSESSIONID").addLogoutHandler((req, res, auth) -> {
```

Comment:
Somewhat magical string. I think the name of the session cookies can be fished up out of the ServletContext but I don't see that available here.